Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM python:3.9-slim
 
 ARG OPENAI_API_KEY
 ARG GOOGLE_CSE_ID
-ARG GOOLGE_API_KEY
```

Comment:
we need to understand the contextualizer better, also saw this and it never created an error?